<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WidgetConvertHeightToNormalMap</class>
 <widget class="QWidget" name="WidgetConvertHeightToNormalMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="cmbType"/>
   </item>
   <item>
    <widget class="QComboBox" name="cmbChannel"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item column="1" row="0" rowspan="3">
      <widget class="KoAspectButton" name="btnAspect" native="true"/>
     </item>
     <item column="0" row="0">
      <widget class="KisDoubleSliderSpinBox" name="sldHorizontalRadius" native="true"/>
     </item>
     <item column="0" row="2">
      <widget class="KisDoubleSliderSpinBox" name="sldVerticalRadius" native="true"/>
     </item>
     <item column="0" row="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>XYZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbRed"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbGreen"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBlue"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header>KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>